packages/app-desktop/bridge.ts

```diff
@@ -288,9 +288,7 @@ export class Bridge {
 	}
 
 	/* returns the index of the clicked button */
-	public showMessageBox(message: string, options: MessageDialogOptions = null) {
-		if (options === null) options = { message: '' };
-
```

Comment:
This is the change from https://github.com/laurent22/joplin/pull/10084 and was added to allow testing the "delete plugin" button. As such, https://github.com/laurent22/joplin/pull/10084 should be reviewed before this PR.

Reply:
Sorry I missed that comment, I'm going to check that PR next then